### PR TITLE
Show video instructions on ATLAS appointments, refactor video visit logic

### PR DESCRIPTION
**Pull request by jbalboni**
## Description
This PR
- Updates the video logic so that instructions show up on ATLAS and at home appointments
- Some minor updates to how past appointments display, mainly around hiding some explanatory text and changing the tense of a heading
- Refactors the video components to be more consolidated and readable, with less hard to follow branching

## Testing done
Local and unit testing

## Screenshots
Screen Shot 2021-06-07 at 1 03 49 PM
Screen Shot 2021-06-07 at 10 54 05 AM
Screen Shot 2021-06-07 at 10 53 48 AM
Screen Shot 2021-06-07 at 10 53 28 AM

## Acceptance criteria
- [ ] Instructions are shown video appointments work as expected

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by alexandec — APPROVED** [review]

File: src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx [hunk 1]
```diff
@@ -32,78 +32,54 @@ export default function VideoVisitLocation({ appointment, facility }) {
...
-
-  if (appointment.vaos.isPastAppointment && isAtlas) {
-    return <AtlasLocation appointment={appointment} />;
-  }
-
-  if (appointment.vaos.isPastAppointment) {
-    return <span>Video conference</span>;
-  }
+    !!appointment.comment &&
+    (isHome || isAtlas) &&
```
Comment (jbalboni) [inline src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/VideoVisitSection.jsx:36]: This fixes the bug

**Review by vbahinwillit — APPROVED** [review]
